diff --git a/Makefile b/Makefile
index 54902d9..dd94c2e 100644
--- a/Makefile
+++ b/Makefile
@@ -113,7 +113,11 @@ endif
 ifeq ($(HOST_OS),Darwin)
   OSX_BUILD := 1
   # Using Homebrew?
-  ifeq ($(shell which brew >/dev/null 2>&1 && echo y),y)
+  ifeq (1,1)
+    CC := gcc
+    CXX := g++
+    CPP := cpp -P
+  else ifeq ($(shell which brew >/dev/null 2>&1 && echo y),y)
 	PLATFORM := $(shell uname -m)
 	OSX_GCC_VER = $(shell find `brew --prefix`/bin/gcc* | grep -oE '[[:digit:]]+' | sort -n | uniq | tail -1)
 	CC := gcc-$(OSX_GCC_VER)
@@ -463,7 +467,7 @@ ENDIAN_BITWIDTH := $(BUILD_DIR)/endian-and-bitwidth
 AS := $(CROSS)as
 
 ifeq ($(OSX_BUILD),1)
-AS := i686-w64-mingw32-as
+AS := arm-none-eabi-as
 endif
 
 ifneq ($(TARGET_WEB),1) # As in, not-web PC port
@@ -492,9 +496,6 @@ ifeq ($(WINDOWS_BUILD),1) # fixes compilation in MXE on Linux and WSL
   CPP := cpp -P
   OBJCOPY := objcopy
   OBJDUMP := $(CROSS)objdump
-else ifeq ($(OSX_BUILD),1)
-  OBJDUMP := i686-w64-mingw32-objdump
-  OBJCOPY := i686-w64-mingw32-objcopy
 else # Linux & other builds
   CPP := $(CROSS)cpp -P
   OBJCOPY := $(CROSS)objcopy
