diff --git a/Makefile b/Makefile
index 54902d9..a3537ae 100644
--- a/Makefile
+++ b/Makefile
@@ -112,26 +112,6 @@ endif
 # macOS overrides
 ifeq ($(HOST_OS),Darwin)
   OSX_BUILD := 1
-  # Using Homebrew?
-  ifeq ($(shell which brew >/dev/null 2>&1 && echo y),y)
-	PLATFORM := $(shell uname -m)
-	OSX_GCC_VER = $(shell find `brew --prefix`/bin/gcc* | grep -oE '[[:digit:]]+' | sort -n | uniq | tail -1)
-	CC := gcc-$(OSX_GCC_VER)
-	CXX := g++-$(OSX_GCC_VER)
-	CPP := cpp-$(OSX_GCC_VER) -P
-	PLATFORM_CFLAGS := -I $(shell brew --prefix)/include
-	PLATFORM_LDFLAGS := -L $(shell brew --prefix)/lib
-  # Using MacPorts?
-  else ifeq ($(shell test -d /opt/local/lib && echo y),y)
-	OSX_GCC_VER = $(shell find /opt/local/bin/gcc* | grep -oE '[[:digit:]]+' | sort -n | uniq | tail -1)
-	CC := gcc-mp-$(OSX_GCC_VER)
-	CXX := g++-mp-$(OSX_GCC_VER)
-	CPP := cpp-mp-$(OSX_GCC_VER) -P
-	PLATFORM_CFLAGS := -I /opt/local/include
-	PLATFORM_LDFLAGS := -L /opt/local/lib
-  else
-	$(error No suitable macOS toolchain found, have you installed Homebrew?)
-  endif
 endif
 
 ifneq ($(TARGET_BITS),0)
@@ -463,7 +443,7 @@ ENDIAN_BITWIDTH := $(BUILD_DIR)/endian-and-bitwidth
 AS := $(CROSS)as
 
 ifeq ($(OSX_BUILD),1)
-AS := i686-w64-mingw32-as
+AS := i586-pc-msdosdjgpp-as
 endif
 
 ifneq ($(TARGET_WEB),1) # As in, not-web PC port
@@ -493,8 +473,9 @@ ifeq ($(WINDOWS_BUILD),1) # fixes compilation in MXE on Linux and WSL
   OBJCOPY := objcopy
   OBJDUMP := $(CROSS)objdump
 else ifeq ($(OSX_BUILD),1)
-  OBJDUMP := i686-w64-mingw32-objdump
-  OBJCOPY := i686-w64-mingw32-objcopy
+  CPP := $(CROSS)cpp -P
+  OBJDUMP := i586-pc-msdosdjgpp-objdump
+  OBJCOPY := i586-pc-msdosdjgpp-objcopy
 else # Linux & other builds
   CPP := $(CROSS)cpp -P
   OBJCOPY := $(CROSS)objcopy
